layouts/includes/head.cfm

```diff
@@ -1,7 +1,7 @@
 <cfoutput>
 	<!---css --->
 	<link href="#event.getModuleRoot( "cbdebugger" )##html.elixirPath(
-		fileName : "css/App.css",
+		fileName : "css/Cbdebugger.css",
```

Comment:
can you use all lowercase `cbdebugger.css`

Reply:
Agreed. I like lowercase as well.